### PR TITLE
Restore internal to external token exchange documentation

### LINKED ISSUE #1745: Incomplete documentation for token-exchange
### Description

In the documentation for the current Keycloak version (20.0.2), the documentation for the token-exchange in "Securing Applications and Services" is incomplete. In comparison to the documentation for version 19.0.3, section 7.3.1 is incomplete and section 7.3.2 is missing. The missing part starts after the third step in section 7.3.1 and after that only the last paragraph of section 7.3.2 is present.

**Docs 20.0.3**: https://www.keycloak.org/docs/20.0.3/securing_apps/index.html#internal-token-to-external-token-exchange
**Docs 19.0.3**: https://www.keycloak.org/docs/19.0.3/securing_apps/index.html#internal-token-to-external-token-exchange

### PR DESCRIPTION
**Pull request by SRTigers98**
Restored internal to external token exchange documentation from commit 0d9a22e643cc0b6db869fd7943625bbaefa5cf92 

Closes #1745

### CODE REVIEW

**Review by andymunro — DISMISSED** [review]

File: securing_apps/topics/token-exchange/token-exchange.adoc [hunk 1]
```diff
@@ -260,6 +260,56 @@ This setup page appears.
 image:images/exchange-idp-permission-setup.png[Identity Provider Exchange Permission Setup]
 
 . Click *Client details* in the breadcrumbs at the top of the screen.
+
+. Click *Policies* tab to create a client policy.
++
+.Client Policy Creation
+image:{project_images}/exchange-idp-client-policy.png[Client Policy Creation]
```
Comment (ahus1) [inline securing_apps/topics/token-exchange/token-exchange.adoc]:
```suggestion
image:images/exchange-idp-client-policy.png[Client Policy Creation]
```

File: securing_apps/topics/token-exchange/token-exchange.adoc [hunk 2]
```diff
@@ -260,6 +260,56 @@ This setup page appears.
...
+
+. Enter the starting client that is the authenticated client that is requesting a token exchange.
+
+. Return to the identity provider's *token-exchange* permission and add the client policy you just defined.
++
+.Apply Client Policy
+image:{project_images}/exchange-idp-apply-policy.png[Apply Client Policy]
```
Comment (ahus1) [inline securing_apps/topics/token-exchange/token-exchange.adoc]:
```suggestion
image:images/exchange-idp-apply-policy.png[Apply Client Policy]
```

**Review by ahus1 — CHANGES_REQUESTED** [review]
Thanks for restoring the parts that have been removed by error. 

Part of that change was also use `image` instead of `{project_images}`, so I ask you to apply this change to your pull request as well, as I assume images will otherwise not show in the built documentation.

When you've updated the docs, please ensure to squash the commits in this PR to a single commit. Thanks!

**Review by ahus1 — APPROVED** [review]
Thank you for restoring this content!

**Comment by ahus1** [comment]
To the maintainers merging this: The previous commit 0d9a22e643cc0b6db869fd7943625bbaefa5cf92 was supposed to remove content that was "product only" inside of `ifeval::[{project_product}==true])]` block. 

The content that's being restored here wasn't adjacent to such a product only block, but removed by accident, and is now being restored by this commit as it is still valid.

**Review by stianst — APPROVED** [review]

**Pull request merged**